ImageSharp.sln

```diff
@@ -1,47 +1,358 @@
+		tests\Images\Input\Bmp\pal8offs.bmp = tests\Images\Input\Bmp\pal8offs.bmp
+		tests\Images\Input\Bmp\pal8os2sp.bmp = tests\Images\Input\Bmp\pal8os2sp.bmp
+		tests\Images\Input\Bmp\pal8os2v1_winv2.bmp = tests\Images\Input\Bmp\pal8os2v1_winv2.bmp
+		tests\Images\Input\Bmp\pal8os2v2-16.bmp = tests\Images\Input\Bmp\pal8os2v2-16.bmp
+		tests\Images\Input\Bmp\pal8os2v2.bmp = tests\Images\Input\Bmp\pal8os2v2.bmp
+		tests\Images\Input\Bmp\pal8v4.bmp = tests\Images\Input\Bmp\pal8v4.bmp
+		tests\Images\Input\Bmp\pal8v5.bmp = tests\Images\Input\Bmp\pal8v5.bmp
+		tests\Images\Input\Bmp\rgb16-565.bmp = tests\Images\Input\Bmp\rgb16-565.bmp
+		tests\Images\Input\Bmp\rgb16-565pal.bmp = tests\Images\Input\Bmp\rgb16-565pal.bmp
+		tests\Images\Input\Bmp\rgb16.bmp = tests\Images\Input\Bmp\rgb16.bmp
+		tests\Images\Input\Bmp\rgb16bfdef.bmp = tests\Images\Input\Bmp\rgb16bfdef.bmp
+		tests\Images\Input\Bmp\rgb24.bmp = tests\Images\Input\Bmp\rgb24.bmp
+		tests\Images\Input\Bmp\rgb32.bmp = tests\Images\Input\Bmp\rgb32.bmp
+		tests\Images\Input\Bmp\rgb32bf.bmp = tests\Images\Input\Bmp\rgb32bf.bmp
+		tests\Images\Input\Bmp\rgb32bfdef.bmp = tests\Images\Input\Bmp\rgb32bfdef.bmp
+		tests\Images\Input\Bmp\rgba32-1010102.bmp = tests\Images\Input\Bmp\rgba32-1010102.bmp
+		tests\Images\Input\Bmp\rgba32.bmp = tests\Images\Input\Bmp\rgba32.bmp
+		tests\Images\Input\Bmp\rgba32abf.bmp = tests\Images\Input\Bmp\rgba32abf.bmp
+		tests\Images\Input\Bmp\rgba32h56.bmp = tests\Images\Input\Bmp\rgba32h56.bmp
+		tests\Images\Input\Bmp\RunLengthEncoded-inverted.bmp = tests\Images\Input\Bmp\RunLengthEncoded-inverted.bmp
+		tests\Images\Input\Bmp\RunLengthEncoded.bmp = tests\Images\Input\Bmp\RunLengthEncoded.bmp
+		tests\Images\Input\Bmp\test16-inverted.bmp = tests\Images\Input\Bmp\test16-inverted.bmp
+		tests\Images\Input\Bmp\test16.bmp = tests\Images\Input\Bmp\test16.bmp
+		tests\Images\Input\Bmp\test8-inverted.bmp = tests\Images\Input\Bmp\test8-inverted.bmp
+		tests\Images\Input\Bmp\test8.bmp = tests\Images\Input\Bmp\test8.bmp
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Gif", "Gif", "{EE3FB0B3-1C31-41E9-93AB-BA800560A868}"
+	ProjectSection(SolutionItems) = preProject
+		tests\Images\Input\Gif\base_1x4.gif = tests\Images\Input\Gif\base_1x4.gif
+		tests\Images\Input\Gif\base_4x1.gif = tests\Images\Input\Gif\base_4x1.gif
+		tests\Images\Input\Gif\cheers.gif = tests\Images\Input\Gif\cheers.gif
+		tests\Images\Input\Gif\giphy.gif = tests\Images\Input\Gif\giphy.gif
+		tests\Images\Input\Gif\kumin.gif = tests\Images\Input\Gif\kumin.gif
+		tests\Images\Input\Gif\leo.gif = tests\Images\Input\Gif\leo.gif
+		tests\Images\Input\Gif\rings.gif = tests\Images\Input\Gif\rings.gif
+		tests\Images\Input\Gif\trans.gif = tests\Images\Input\Gif\trans.gif
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "issues", "issues", "{BF8DFDC1-CEE5-4A37-B216-D3085360C776}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
What's all this stuff? Feels like a relic from the past.

Reply:
These are all files that you currently have checked directly into the repository, namely under `tests\Images`.

Reply:
Can we wildcard this?

Reply:
I don't believe solution files support that, and from a quick test locally it doesn't work.

I also don't feel particularly strongly about this being in here, I've just found that having all files accessible from the solution is convenient.